src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx

```diff
@@ -121,8 +126,51 @@ class CreateSchemeRegistrarProposal extends React.Component<IProps, IState> {
     this.setState({tags});
   }
 
+  public exportFormValues(values: IFormValues) {
+    values = {
+      ...values,
+      ...this.state,
+    };
+    exportUrl(values);
+    this.props.showNotification(NotificationStatus.Success, "Exportable url is now in clipboard :)");
+  }
+  
+  loadCurrentTab(){
+    let initialCurrentTab = importUrlValues({ schemeToAdd: "", schemeToEdit: "", schemeToRemove: ""});
```

Comment:
this seems like a strange way to specify the tab, why do you need three different params which override each other? why not just a single tab=XXX parameter?

Reply:
because this way i am telling importUrlValues to return me only those properties( schemeToAdd, schemeToEdit, schemeToRemove)if found in search url params.

Reply:
But why not just use a property named "tab" like @tibetsprague is suggesting and set it to "addScheme" | "editScheme" | "removeScheme"?